index.js

```diff
@@ -497,14 +501,13 @@ class ClientServiceBase {
         ],
     );
 
-    const signerFactory = new SignerFactory(
-        clientProperties.getPrivateKeyCryptoKey() ||
-        clientProperties.getPrivateKeyPem()
-    );
     const propertiesJson = JSON.stringify(properties);
     const builder = new ContractRegistrationRequestBuilder(
         new this.protobuf.ContractRegistrationRequest(),
-        signerFactory.create(),
+        this._createSigner(
+            clientProperties.getPrivateKeyCryptoKey() ||
```

Comment:
For consistency, 
```suggestion
            properties.getPrivateKeyCryptoKey() ||
```

Reply:
Ah, this one is because the `properties` is used by contract properties.

Reply:
Ah, then let's name the contract properties `contractProperties` and name it `properties` otherwise `properties` is used for different things and a bit confusing.

Reply:
No problem.

Reply:
Revised !